popup.css

```diff
@@ -0,0 +1,142 @@
+.projectPopup {
+  position: fixed;
+  top: 0;
+  z-index: 1;
+  background-color: rgba(193, 199, 208, 0.9);
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.project1 {
+  background-color: #fff;
+  height: 97%;
+  width: 92%;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.hide {
+  display: none;
+}
+
+.popupTop {
+  width: 85%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 15px;
+}
+
+.titlePopup {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.projectImage {
+  display: flex;
+  justify-content: center;
+}
+
+.popupClose {
+  width: 30px;
+  height: 30px;
+  cursor: pointer;
+}
+
+.popupClose:hover {
+  transform: scale(1.2);
+  transition: all 0.7s;
+}
+
+.actionBtnPopup {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.btnPopup {
+  font-size: 0.8rem;
+  height: 38px;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-around;
+  align-items: center;
+  text-decoration: none;
+}
+
+.seeAll {
+  width: 20px;
+  height: 20px;
+}
+
+.popupBtns,
+.popupDown {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+@media only screen and (min-width: 768px) {
+  .project1 {
+    background-color: #fff;
+    height: 96%;
+    width: 90%;
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .popupTop {
+    height: 85%;
+    width: 85%;
+    display: flex;
+    flex-direction: column;
+    gap: 15px;
+  }
```

Comment:
- **[OPTIONAL]** It looks good in the desktop version 👏🏽, however, it is normal that the modal window exceeds the screen in the y-axis I recommend using `overflow-y: scroll;`.

Reply:
Hi @franclobo ,

Thanks for your time and for reviewing my code. I have resolved all the issue. 🙏